opendbc/sunnypilot/car/chrysler/carcontroller_ext.py

```diff
@@ -0,0 +1,17 @@
+from opendbc.car import structs
+from opendbc.car.interfaces import CarStateBase
+from opendbc.car.chrysler.values import RAM_DT
+
+class CarControllerExt:
+  def __init__(self, CP: structs.CarParams):
+    self.CP = CP
+
+  def get_lkas_control_bit(self, CS: CarStateBase, lkas_control_bit: bool, lkas_control_bit_prev: bool) -> bool:
+    if self.CP.carFingerprint in RAM_DT:
+      lkas_control_bit = lkas_control_bit_prev
+      if self.CP.minEnableSpeed <= CS.out.vEgo <= self.CP.minEnableSpeed + 0.5:
+        lkas_control_bit = True
+      if (self.CP.minEnableSpeed >= 14.5) and (CS.out.gearShifter != 2):
```

Comment:
Per this line, `self.CP.minEnableSpeed` is already `14.5` for Ram 1500, do we really need to have another check with `self.CP.minEnableSpeed >= 14.5`?
https://github.com/dparring/opendbc/blob/468b7313615105605a51997bab8255299d331028/opendbc/car/chrysler/interface.py#L87

```suggestion
      if CS.out.gearShifter != 2:
```

Reply:
agree, it is unnecessary